### PR TITLE
Configured jest for testing, added src dir with index.html and index.js

### PR DESCRIPTION
**Pull request by Lana2683**
* Added .gitignore;
* Added index.html;
*  Configured jest

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created by https://www.toptal.com/developers/gitignore/api/react
```
Comment (mmarkelov) [inline .gitignore:1]: You should also remove `node_modules` from your repo.

File: package.json [hunk 2]
```diff
@@ -14,28 +14,30 @@
...
     "react-redux": "^7.2.2",
     "react-router": "^5.2.0",
     "redux": "^4.0.5",
-    "webpack": "^5.23.0",
-    "webpack-dev-server": "^3.11.2"
+    "webpack": "^5.23.0"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```
Comment (mmarkelov) [inline package.json]: Could you please add some init tests?

File: .gitignore (hunk 1 above)
Comment (Lana2683) [reply .gitignore:1]: Fixed

File: package.json (hunk 2 above)
Comment (Lana2683) [reply package.json]: fixed

**Pull request merged**